### DISCUSSION TITLE
Change type of scales on maps

### DISCUSSION THREAD

**Discussion by LucyGwilliamAdmin** [open-sdg/open-sdg#1447] [Ideas]
**Is your feature request related to a problem? Please describe.**
Currently maps show equal-sized intervals in scale, for example see [here](https://sdgdata.gov.uk/16-6-2/):
![image](https://user-images.githubusercontent.com/43404358/93365520-cd59d280-f841-11ea-851f-428ae27a7e71.png)

In most cases data is not distributed evenly, so it's not correct to have equal-sized intervals

**Describe the solution that you need**
Instead scale should be representative of how data is distributed like in this map shown [here](https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/conditionsanddiseases/articles/wheredoover70slive/2020-03-24):

![image](https://user-images.githubusercontent.com/43404358/93365767-1f9af380-f842-11ea-806d-97b15d537885.png)

**Why do you need this feature?**
Recommended as best practice by Geography expert at ONS

**Describe alternatives you've considered**
None

**Comment by WohnungNDS** [comment]
Perhaps has to be seen in conjunction with #1217
